### ISSUE TITLE
Uncaught TypeError: Argument 1 passed to App\Exceptions\Handler::report() must be an instance of Exception, instance of TypeError given

### ISSUE THREAD

**Issue by bunlongheng** [vinkla/instagram#71]
I kept getting this issue after installing this in my Laravel 5.1 project.

![2018-02-27 at 1 55 36 pm](https://user-images.githubusercontent.com/11523064/36748618-5fbc6ecc-1bc6-11e8-88cc-58f7f30c333c.png)

I followed everything in the instruction. 

I am on Mac OS X, PHP 7.1, Laravel 5.1 

**How would one go about and debug this further ?** 

_______________ 

I'm open to any suggestions at this moment. 

Any hints/suggestions / helps on this be will be much appreciated!

**Comment by vinkla** [comment]
This seems to be an issue with your project and not this library. Please ask in Laracasts forum or in the Laravel Slack workspace.

**Comment by bunlongheng** [comment]
That’s not true.

I am using Laravel 5.1, and my project working fine, until, I integrate your packages in and I see a crash.


On Feb 27, 2018, at 3:00 PM, Vincent Klaiber <notifications@github.com<mailto:notifications@github.com>> wrote:


This seems to be an issue with your project and not this library. Please ask in Laracasts forum or in the Laravel Slack workspace.

**Comment by vinkla** [comment]
This could still be an issue with your project. The error message you have provided doesn’t say anything about this package.

**Comment by bunlongheng** [comment]
You said, we need to have PHP version 7.1 or 7.2.

I have 7.1.4

I followed everything you said in the repo, boom, refresh my app.
I got that.

That is something wrong with the package.
It expects something base on different PHP version or Laravel version.


Share your opinion if you disagree.


On Feb 27, 2018, at 3:11 PM, Vincent Klaiber <notifications@github.com<mailto:notifications@github.com>> wrote:


This could still be an issue with your project. The error message you have provided doesn’t say anything about this package.
